crypto/keys/pem_signer_factory.go

```diff
@@ -21,17 +21,32 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"fmt"
+	"sync"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 	"github.com/google/trillian"
 	"github.com/google/trillian/crypto/keyspb"
 )
 
+var (
+	pkcs11Module string
```

Comment:
The main reason for not doing this is that it requires adding a method to set it to the `SignerFactory` interface such that other implementations would need to add a dummy method to fulfill the interface. That said if you think it would be cleaner to do it that way I'm happy to make the change.